blocks/video/src/components/media.tsx

```diff
@@ -189,108 +119,86 @@ export const Media: FunctionComponent<
     };
   }, []);
 
+  const propertiesRef = useRef(properties);
+
+  useLayoutEffect(() => {
+    propertiesRef.current = properties;
+  });
+
   const updateData = useCallback(
-    ({ width, src }: { src: string | undefined; width?: number }) => {
-      if (readonly) {
-        return;
-      }
+    ({ src }: { src: string | undefined; }) => {
       if (src?.trim()) {
-        if (graphService?.updateEntity && entityId) {
+        // @todo how to handle this not being defined now
```

Comment:
Shall we track this in a follow-up or does it need to be addressed now?

Reply:
Comment is out of date, we can just remove the check